vllm/model_executor/layers/quantization/modelopt.py

```diff
@@ -1464,7 +1466,9 @@ def process_weights_after_loading(self, layer: torch.nn.Module) -> None:
         layer.w13_weight_scale_2 = Parameter(w13_weight_scale_2, requires_grad=False)
 
         # Common processing for input scales and alphas
-        w13_input_scale = layer.w13_input_scale.max(dim=1).values.to(torch.float32)
+        w13_input_scale = (
+            layer.w13_input_scale.max().to(torch.float32).expand(layer.num_experts)
+        )
```

Comment:
Why are we performing a max and then running an expand? Also, w13_input_scale.max(dim=1) takes a max of w1 and w3 and generates [E,] where as 
layer.w13_input_scale.max() takes the global max of the stacked tensor? And additionally, why do we want to change this for all paths?

Reply:
The scope should be limited to modelopt_nvfp4 quantization only.
To ensure consistency in subsequent multiplication dimensions, it’s preferable to preserve the original tensor shape as before this PR, while changing only the numerical behavior.

Reply:
sure. Will make the change.